exercises/circular-buffer/circular-buffer.js

```diff
@@ -0,0 +1,35 @@
+//
+// This is only a SKELETON file for the 'Circular Buffer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+class CircularBuffer {
+  write() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  read() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  forceWrite() {
+    throw new Error("Remove this statement and implement this function");
+  }
+}
+
+export default () => {
+  // Complete this with the code for the exercise.
+  return new CircularBuffer();
+}
```

Comment:
We can directly export the class itself.

```js
export default CircularBuffer;
```

Reply:
I had thought about this, but if I do, when running the tests I see this error:
` TypeError: Class constructor CircularBuffer cannot be invoked without 'new'`

      3 | describe('CircularBuffer', () => {
      4 |   test('reading an empty buffer throws a BufferEmptyError', () => {
    > 5 |     const buffer = circularBuffer(1);
        |                    ^
      6 |     expect(() => buffer.read()).toThrow(BufferEmptyError);
      7 |   });
      8 | 

      at Object.test (exercises/circular-buffer/circular-buffer.spec.js:5:20)

The way it is used from the spec file made me think that it would need to export a function. What do you think? 
If this error is acceptable, I can change it to default export the class.

Reply:
Test files should use `new CircularBuffer`.

Keep `export default CircularBuffer` in stub file & `const buffer = new CircularBuffer` in test file.

Reply:
Oh, my bad, I didn't change it, but I hadn't updated it and it was modified a couple of days ago. I'll update it in a few minutes.